eng/targetingpacks.targets

```diff
@@ -6,6 +6,33 @@
     <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
   </PropertyGroup>
 
+  <!-- Add Known* items if the SDK doesn't support the TargetFramework yet. -->
+  <ItemGroup Condition="'$(_UseLocalTargetingRuntimePack)' == 'true'">
+    <KnownFrameworkReference Include="$(SharedFrameworkName)"
+                             DefaultRuntimeFrameworkVersion="$(ProductVersion)"
+                             IsTrimmable="true"
+                             LatestRuntimeFrameworkVersion="$(ProductVersion)"
+                             RuntimeFrameworkName="$(SharedFrameworkName)"
+                             RuntimePackNamePatterns="$(SharedFrameworkName).Runtime.**RID**"
+                             RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm"
+                             TargetFramework="$(NetCoreAppCurrent)"
+                             TargetingPackName="$(SharedFrameworkName).Ref"
+                             TargetingPackVersion="$(ProductVersion)"
+                             Condition="'@(KnownFrameworkReference)' != '' and !@(KnownFrameworkReference->AnyHaveMetadataValue('TargetFramework', '$(NetCoreAppCurrent)'))" />
+    <KnownAppHostPack Include="$(SharedFrameworkName)" 
+                      AppHostPackNamePattern="$(SharedFrameworkName).Host.**RID**"
+                      AppHostPackVersion="$(MicrosoftNETCoreAppVersion)"
```

Comment:
I guess this doesn't use ProductVersion as that one doesn't use the live built asset?

Reply:
nope, that was just a mistake. Fixed. Thanks